middleware/queue/source/group/handle.cpp

```diff
@@ -229,6 +229,7 @@ namespace casual
                         catch( ...)
                         {
                            log::line( log::category::error, exception::capture(), " failed with enqueue request to queue: ", message.name);
+                           state.multiplex.send( message.process.ipc, common::message::reverse::type( message));
```

Comment:
I had expected some kind of error code to be set to indicate what happened. Appears as if there is no way to return an error in the reply, except by not providing a "quid" in message.queue of the reply.